testing/runner.ts

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env deno -A
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { parse } from "../flags/mod.ts";
+import { glob, walk } from "../fs/mod.ts";
+import { runTests } from "./mod.ts";
+const { args, cwd } = Deno;
+
+const DEFAULT_GLOBS = [
+  "**/*_test.ts",
+  "**/*_test.js",
+  "**/test.ts",
+  "**/test.js"
+];
+
+/* eslint-disable max-len */
+function showHelp(): void {
+  console.log(`Deno test runner
+
+USAGE:
+  deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [FILES...]
+
+OPTIONS:
+  -q, --quiet               Don't show output from test cases 
+  -f, --failfast            Stop test suite on first error
+  -e, --exclude <FILES...>  List of file names to exclude from run. If this options is 
+                            used files to match must be specified after "--". 
```

Comment:
[`--testPathIgnorePatterns=[array]`](https://jestjs.io/docs/en/cli#testpathignorepatterns-array)

> An array of regexp pattern strings that is tested against all tests paths before executing the test. Contrary to `--testPathPattern`, it will only run those test with a path that does not match with the provided regexp expressions.

Reply:
Also, I’m wondering if `-x` would be a better short name for this option if you keep the `exclude` name.